internal/commands/relationship.go

```diff
@@ -124,57 +127,67 @@ func bulkDeleteRelationships(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	request := &v1.ReadRelationshipsRequest{RelationshipFilter: filter}
-
-	counter := -1
-	if cobrautil.MustGetBool(cmd, "estimate-count") {
-		request.Consistency = &v1.Consistency{Requirement: &v1.Consistency_FullyConsistent{FullyConsistent: true}}
+	bar := console.CreateProgressBar("deleting relationships")
+	defer func() {
+		_ = bar.Finish()
+	}()
 
-		ctx, cancel := context.WithCancel(cmd.Context())
-		defer cancel()
+	allowPartialDeletions := cobrautil.MustGetBool(cmd, "force")
+	optionalLimit := cobrautil.MustGetUint(cmd, "optional-limit")
+	var resp *v1.DeleteRelationshipsResponse
+	for {
+		delRequest := &v1.DeleteRelationshipsRequest{
+			RelationshipFilter:            filter,
+			OptionalLimit:                 uint32(optionalLimit),
```

Comment:
add a bounds check here

Reply:
addressed in https://github.com/authzed/zed/pull/365/commits/810734aa333c96bd9b7437c3965579293777885e